### PR TITLE
Remove scsi files

### PR DESCRIPTION
**Pull request by Vikort**
Fixed scsi files for build kb

### CODE REVIEW

File: prepare_kb.py [hunk 1]
```diff
@@ -0,0 +1,10 @@
+import os
+from scripts.copy_kb import copy_kb
+
+kb_path = os.path.dirname(os.path.abspath(__file__))
+ostis_path = os.path.split(kb_path)[0]
+copy_path = os.path.join(ostis_path, 'ims.ostis.kb_copy')
+copy_kb(kb_path, copy_path)
+for script in os.listdir(os.path.join(kb_path, 'scripts')):
+    if os.path.isfile(os.path.join(kb_path, 'scripts', script)) and script != 'copy_kb.py' and script.endswith('.py'):
+        os.system("python3 " + os.path.join(kb_path, 'scripts', script) + ' ' + copy_path)
```
Comment (KovalM) [inline prepare_kb.py:10]: I think we will need to have ability to run scripts in a given order or not run some script. Simple way do support it is create list of scripts here and run each script from this list in turn

**Review by KovalM — APPROVED** [review]

File: scripts/remove_scsi.py [hunk 2]
```diff
@@ -0,0 +1,58 @@
+import sys
+import os
+
+
+def main(work_directory):
+    if os.name == 'nt':
```
Comment (MikhailSadovsky) [inline scripts/remove_scsi.py:6]: Please remove this if section as we're not supporting nt

File: prepare_kb.py [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+class CopyKbPaths(Enum):
+    KB = os.path.dirname(os.path.abspath(__file__))
+    OSTIS = os.path.split(KB)[0]
+    KB_COPY = os.path.join(OSTIS, 'ims.ostis.kb_copy')
+    SCRIPTS = os.path.join(KB, 'scripts')
+
+copy_kb(CopyKbPaths.KB.value, CopyKbPaths.KB_COPY.value)
+scripts = [os.path.join(CopyKbPaths.SCRIPTS.value, 'remove_scsi.py')]
+for script in scripts:
+    if os.path.isfile(script) and script != 'copy_kb.py' and script.endswith('.py'):
```
Comment (MikhailSadovsky) [inline prepare_kb.py:15]: Looks like this condition isn't needed as `scripts` list created by hands

File: scripts/remove_scsi.py [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+                with open(file, "w", encoding="utf_8") as scs_file:
+                    for scs_file_line in buffer_file_list:
+                        scs_file.write(scs_file_line)
+
+
+if __name__ == '__main__':
+    if len(sys.argv) == 2:
+        main(sys.argv[1])
+    else:
+        print("invalid numb of arguments, Please specify only the work directory")
```
Comment (MikhailSadovsky) [inline scripts/remove_scsi.py:58]:
```suggestion
        print("invalid number of arguments, Please specify only the work directory")

```

File: scripts/remove_scsi.py [hunk 5]
```diff
@@ -0,0 +1,58 @@
...
+    if os.name == 'nt':
+        os.system('reg delete "HKLM\SYSTEM\CurrentControlSet\Control\FileSystem" /v LongPathsEnabled /f')
+        os.system('reg add "HKLM\SYSTEM\CurrentControlSet\Control\FileSystem" /v LongPathsEnabled /t REG_DWORD /d 1')
+
+    list_of_files = list()
+    for (dir_path, dir_names, file_names) in os.walk(work_directory):
+        list_of_files += [os.path.join(dir_path, file) for file in file_names]
+    for file in list_of_files:
+        if file.endswith(".scs"):
+            is_edit = False
```
Comment (MikhailSadovsky) [inline scripts/remove_scsi.py:15]:
```suggestion
            is_file_processed = False

```

**Review by MikhailSadovsky — CHANGES_REQUESTED** [review]

**Comment by MikhailSadovsky** [comment]
Please add the brief description of your changes in PR description

File: prepare_kb.py [hunk 6]
```diff
@@ -0,0 +1,17 @@
...
+class CopyKbPaths(Enum):
+    KB = os.path.dirname(os.path.abspath(__file__))
+    OSTIS = os.path.split(KB)[0]
+    KB_COPY = os.path.join(OSTIS, 'ims.ostis.kb_copy')
+    SCRIPTS = os.path.join(KB, 'scripts')
+
+copy_kb(CopyKbPaths.KB.value, CopyKbPaths.KB_COPY.value)
+scripts = [os.path.join(CopyKbPaths.SCRIPTS.value, 'remove_scsi.py')]
+for script in scripts:
+    # if os.path.isfile(script) and script != 'copy_kb.py' and script.endswith('.py'):
```
Comment (MikhailSadovsky) [inline prepare_kb.py:15]:
```suggestion
```

**Review by MikhailSadovsky — CHANGES_REQUESTED** [review]

**Comment by MikhailSadovsky** [comment]
Closing as a duplicate for #91

**Pull request closed without merging**